0400-methods.Rmd

```diff
@@ -0,0 +1,41 @@
+# Methods
+
+This repo is designed as a place for default projects as well as templates and scripts required for data extraction and application. Also included are descriptions of workflows utilising Mergin Maps in QGIS to collect data in the field for fish passage assessment, habitat assessment and stream crossing climate change/maintenance issue risk assessment.
+
+<br>
+
+Mergin Maps is an open source GIS integration software developped by Lutra Consulting. It uses a plugin found in QGIS and communicates with an associated mobile app to make field data collection even easier. A link to their website with helpful documentation and a place to sign up for a free account can be found [here](https://merginmaps.com/?gclid=Cj0KCQjwnvOaBhDTARIsAJf8eVPAtSEg0pF915dFv6b4kge16gvwTRnX75S0gEB4Xj2wRZevorN3n3EaAhSrEALw_wcB). Note that free accounts only come with 100mb storage capacity to start. However, projects shared with you from other individuals do not take up storage space on your account. In the 2022 field season, we carried out a pilot project where we utilised mergin to collect our fish passage data through a digital interface.
+
+How to build a mergin project:
+
+-   Run bash script located `scripts/background_layers.sh` on command line. This script can be customised depending on the area of interest.
```

Comment:
why not insert all the text I put in slack about the tools that `scripts/background_layers.sh` uses (bcdata, wget, fwapg)?  I wrote it so you could leverage it

Reply:
There is a little section on that in the readme file. But I can add it here too if that makes sense.